runtime/metricsview/executor_rewrite_time.go

```diff
@@ -41,6 +73,47 @@ func (e *Executor) resolveTimeRange(ctx context.Context, tr *TimeRange, tz *time
 		return nil
 	}
 
+	if tr.RillTime == "" {
+		return e.resolveTimeRangeLegacy(ctx, tr, tz, executionTime)
+	}
+
+	rt, err := rilltime.Parse(tr.RillTime)
```

Comment:
Can we avoid calling this `rt` – we often use that abbreviation for the `Runtime` object, e.g. `e.rt` in this context is the `Runtime` object. So it's a bit too confusing.